examples/cem_cartpole.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+"""
+This is an example to train a task with Cross Entropy Method.
+
+Here it runs CartPole-v1 environment with 100 epoches.
+
+Results:
+    AverageReturn: 100
+    RiseTime: epoch 8
+"""
+from garage.algos import CEM
+from garage.baselines import LinearFeatureBaseline
+from garage.experiment import LocalRunner, run_experiment
+from garage.tf.envs import TfEnv
+from garage.tf.policies import CategoricalMLPPolicy
+from garage.tf.samplers import OnPolicyVectorizedSampler
+
+
+def run_task(*_):
+    """Train CEM with Cartpole-v1 environment."""
+    with LocalRunner() as runner:
+        env = TfEnv(env_name="CartPole-v1")
+
+        policy = CategoricalMLPPolicy(
+            name="policy", env_spec=env.spec, hidden_sizes=(32, 32))
+
+        baseline = LinearFeatureBaseline(env_spec=env.spec)
+
+        runner.initialize_tf_vars()
+
+        n_samples = 20
+
+        algo = CEM(
+            env_spec=env.spec,
+            policy=policy,
+            baseline=baseline,
+            best_frac=0.1,
+            max_path_length=100,
+            n_samples=n_samples)
+
+        runner.setup(algo, env, sampler_cls=OnPolicyVectorizedSampler)
+        # NOTE: make sure that n_epoch_cycles == n_samples !
```

Comment:
why?

Reply:
Because CEM samples one policy per epoch cycle, and only update policy distribution at the end of the epoch. `n_samples` specifies the number of policies to be sampled in one epoch.

Reply:
But I think there should be a way to verify this. I just can't figure how to do it.

Reply:
another way to impose this constraint is to subclass Runner just for CEM and override setup()

Reply:
Can we rename n_samples to policy_update_freq? I think n_samples is a bit confusing.

Reply:
@ryanjulian But this will require picking up an algorithm-specific runner. 
Maybe a simpler way is to do it without subclassing, by checking if the algo is CEM and do verify.

Reply:
@CatherineSue I think `n_samples` is more natural for people familar with CEM, because it is defined as is in the literature.

Reply:
@naeioi how would that work if CEM and the Runner were not in the same codebase?

You've already defined an algorithm-specific sampler. You just did it in a comment, rather than in code :stuck_out_tongue:

Reply:
FWIW it's up to you. Feel free to merge.